src/Quote.js

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from "react";
+import { New } from "./New.js";
+
+export function Quote({
+  color, 
+  loading, 
+  error,
+  customError, 
+  data,
+  handleClick
+}) {
+  return(
+    <Fragment>
+
+      { loading && 
+        <div id="loading-box">
+          <img 
+              class="icon"
+              src="images/loader.gif" 
+              alt="Eye glasses' arms opening and closing with eyes roving and blinking" 
+            />
+          <p class="primary-text">Quote incoming!</p>
+        </div>
+      }
+
+      { error && 
+        <div>
+          <img 
+              class="icon"
+              src="images/error.png" 
+              alt="Triangle with '!' inside to indicate error" 
+            />
+          <p class="primary-text">{error}</p>
+          <p className="custom-message">{customError}</p>
+          
+          <New 
```

Comment:
First place `New` component is called. Its styling is complete with just its `id` (defined in New.js), so now `className` is called here.